sparkle/src/stories/MultiPageSheet.stories.tsx

```diff
@@ -455,6 +455,7 @@ export const WithConditionalNavigation: Story = {
           disableNext={
             currentPageId === "data-selection" && selectedItems.length === 0
           }
+          disableSave={!description.trim()}
```

Comment:
why trim? to disallow a description contains only spaces or something? 🤔

Reply:
Yeah quite random, it's just a toy example in storybook